lib/matplotlib/backends/backend_agg.py

```diff
@@ -563,23 +563,25 @@ def print_jpg(self, filename_or_obj, *args, dryrun=False, pil_kwargs=None,
             `PIL.Image.save` when saving the figure.  These take precedence
             over *quality*, *optimize* and *progressive*.
         """
-        FigureCanvasAgg.draw(self)
+        # The image is "pasted" onto a white background to handle transparency.
```

Comment:
I think the term "paste" is a bit misleading

```suggestion
        # Remove transparency by alpha-blending on an assumed white background
```

Reply:
sure, sounds good.